src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTupleTest.cs

```diff
@@ -4402,7 +4402,7 @@ public ValueTuple(T1 item1, T2 item2)
             Assert.Same(mTuple, mItem1.ContainingSymbol);
             Assert.True(mItem1.CustomModifiers.IsEmpty);
             Assert.True(mItem1.GetAttributes().IsEmpty);
-            Assert.Equal("error CS8128: Member 'Item1' was not found on type 'ValueTuple<T1, T2>' from assembly 'comp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.",
+            Assert.Equal($"error CS8128: {string.Format(CSharpResources.ERR_PredefinedTypeMemberNotFoundInAssembly, "Item1", "ValueTuple<T1, T2>", "comp, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null")}",
                          mItem1.GetUseSiteDiagnostic().ToString());
```

Comment:
>ToString [](start = 55, length = 8)

Please call overload that takes IFormatProvider instead and pass english culture, we have a helper that we use in tests to get that culture EnsureEnglishUICulture.PreferredOrNull.